terragrunt/aws/iam/iam.tf

```diff
@@ -7,7 +7,9 @@ data "aws_iam_policy_document" "saas_procurement" {
       identifiers = ["ecs-tasks.amazonaws.com"]
     }
   }
+}
 
+data "aws_iam_policy_document" "saas_procurement_ssm" {
```

Comment:
Nice - one last thing you'll need to do is create a `aws_iam_policy` resource and attach it to the role with a `aws_iam_role_policy_attachment`.  Here's an example:
https://github.com/cds-snc/scan-files/blob/5b153e30dbeb62fef5540125b6b1f95409ebc9ee/terragrunt/aws/api/iam.tf#L172-L186

Reply:
Thanks Pat! I did update my PR to have the a policy and then attach it.